acceptance/indy_acceptance/indy-tests/test_scenarios/test_scenario_03.py

```diff
@@ -0,0 +1,185 @@
+import json
+import sys
+import logging
+import os
+import asyncio
+import shutil
+import time
+from indy import pool, signus, wallet
+from indy.error import IndyError
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from utils.constant import Colors, Constant
+from utils.report import TestReport
+
+
+class MyVars:
+    begin_time = 0
+    pool_handle = 0
+    wallet_handle = 0
+    pool_name = "pool_genesis_test3"
+    wallet_name = "test_wallet3"
+    debug = False
+    test_report = TestReport("Test_scenario_03_Check_Connection")
+    test_results = {"Step 1": False, "Step 2": False, "Step 3": False, "Step 4": False,
+                    "Step 5": False, "Step 6": False, "Step 7": False, "Step 8": False}
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def test_precondition():
```

Comment:
Looks like this code is used in all the tests. Can we create a common method for this?

Reply:
I agree.  Khoi, please make it so